terraform/modules/ops/build.tf

```diff
@@ -3,7 +3,8 @@
 resource "google_cloudbuild_trigger" "api_push_to_main" {
```

Comment:
Do you mind changing the resource names to match the new naming?

Reply:
done